src/Domain.LinnApps/Consignments/ConsignmentService.cs

```diff
@@ -102,6 +103,10 @@ public async Task<ProcessResult> PrintConsignmentDocuments(int consignmentId, in
             var printerUri = this.GetPrinterUri(userNumber);
             var consignment = this.consignmentRepository.FindById(consignmentId);
 
+            await this.
```

Comment:
this method returns a PrintResult - should we not be using the result for  something?